Source/BoundaryConditions/PML_RZ.H

```diff
@@ -0,0 +1,86 @@
+/* Copyright 2019 David Grote
+ *
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+#ifndef WARPX_PML_RZ_H_
+#define WARPX_PML_RZ_H_
+
+#include "PML_RZ_fwd.H"
+
+#ifdef WARPX_USE_PSATD
+#   include "FieldSolver/SpectralSolver/SpectralSolverRZ.H"
+#endif
+
+#include <AMReX_MultiFab.H>
+#include <AMReX_BoxArray.H>
+#include <AMReX_Config.H>
+#include <AMReX_FabArray.H>
+#include <AMReX_FabFactory.H>
+#include <AMReX_GpuContainers.H>
+#include <AMReX_IntVect.H>
+#include <AMReX_REAL.H>
+#include <AMReX_Vector.H>
+
+#include <AMReX_BaseFwd.H>
+
+#include <array>
+#include <memory>
+#include <string>
+#include <vector>
```

Comment:
Same here - I can only make guesses about which includes are needed. If too many are included or if a one is missed (but included indirectly) there are no error messages. I'll double check though by inspection.

Reply:
Thank you, Dave. I wonder if @lucafedeli88 has suggestions on how to check this, maybe using some "include-what-you-used" tool that was used already to clean up the include directives in WarpX? @lucafedeli88 If you have a brief recipe for how to check these include directives systematically, rather than by inspection, please let us know. Thank you!